inabox/tests/integration_v2_test.go

```diff
@@ -182,6 +185,15 @@ func TestEndToEndV2Scenario(t *testing.T) {
 	err = testHarness.StaticCertVerifier.CheckDACert(ctx, cert4)
 	require.NoError(t, err)
 
+	// for brevity, test the ARB proof validator contract which wraps around
+	// the CertVerifierRouter to validate a Custom DA Proof for happy && unhappy path
+
+	arbCertOneStepProof, err := serializeARBVerifyCertProof(cert4)
+	require.NoError(t, err)
+
+	passed, err := testHarness.ArbitrumEigenDAProofValidator.ValidateCertificate(&bind.CallOpts{}, arbCertOneStepProof)
+	require.NoError(t, err)
+	require.True(t, passed)
```

Comment:
could we put this in another arb-specific test? Good to keep tests more self-contained imo. @litt3 has already written a few payments related inabox tests, so should be possible to copy his framework.